[docs] Syntax error on line 217 - (Handle push notifications with React Navigation)

ace-builds opened the pull request:
# Why

 Because of the 'url' variable was declared on line 209 as a constant (const), Line 217 throws a Cannot redeclare block-scoped variable 'url' syntax error because it tries to reassign the variable.

Using 'let' allows the reassignment of the url variable on line 217

# How

I copied the code to my code editor and it showed up

# Test Plan

My app built successfully built without throwing any syntax error

# Checklist

<!--
Please check the appropriate items below if they apply to your diff. This is required for changes to Expo modules.
-->

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

ace-builds committed:
Syntax error on line 217

Because of the 'url' variable declared on line 209 with const, Line 217 throws a Cannot redeclare block-scoped variable 'url' syntax error because it tries to reassign the variable.

Using 'let' would allow a reassignment of the url variable on line 217

Simek reviewed (commented):
Hello @Chinedu19, nice catch, thank you for working on the corrections! 👍 

Can we instead just return the second URL directly, instead of re-assigning it into `url` variable? This would let use avoid `let` refactor.

It would be also nice, if you can backport this correction to the versioned docs, at least for latest SDK (48):
* https://github.com/expo/expo/tree/main/docs/pages/versions

ace-builds committed:
Merge branch 'expo:main' into patch-1

ace-builds committed:
Update to Unversioned docs/notifications.mdx

returned notification data url directly

ace-builds committed:
Update to expo V48.0.0 docs/notifications.mdx

returned notification data url directly. When using react-navigation to handle notifications

Chinedu19 commented:
I just made the changes and also made the corrections to SDK (48)

Simek reviewed (approved):
Thank you for the updates, LGTM, welcome to the community! 👍
